guides/common/modules/proc_configuring-project-with-an-alternate-cname.adoc

```diff
@@ -0,0 +1,18 @@
+[id='configuring-project-with-an-alternate-cname_{context}']
+= Configuring {Project} with an Alternate CNAME
+Use this procedure to configure {Project} with an alternate CNAME. Note that the procedures for users of a default {Project} certificate and custom certificate differ. Note that if you have any hosts registered to {Project}, you must update the `katello-ca-consumer` package on the hosts after regenerating the {Project} SSL certificate.
+
+.For Default {Project} Certificate Users
+
+* If you have installed {Project} with a default {Project} certificate and want to configure {Project} with an alternate CNAME, enter the following command on {Project} to generate a new default {Project} SSL certificate with an additional CNAME.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-installer} --certs-cname _alternate_fqdn_ --certs-reset
+----
+
+* If you have not installed {Project}, you can add the `--certs-cname _alternate_fqdn_` option to the `satellite-maintain` command to install {Project} with an alternate CNAME.
```

Comment:
Wait, `satellite-maintain`? Shouldn't that be `satellite-installer`?

Also, I guess it should be written as variable: `{foreman-installer}`.

Sorry, haven't noticed that yesterday

Reply:
Fixed! Thank you for pointing this out, I do not know what I was thinking when writing it.